Please add I-Mobile support to amp-ad

imobile opened the pull request:
Please add support to amp-ad for the imobile type.
Our CLA: i-mobile Co.,Ltd

Looking forward to hearing from your response!

imobile commented:
Hello we signed the CLA as a corporation, would you please have it double checked?
Our CLA: i-mobile Co.,Ltd

imobile commented:
Thank you very much for confirming! Looking forward to your feedbacks

imobile commented:
hi @cramforce would you please have our repo reviewed? Thanks a lot.

imobile commented:
hi @ilovelili could you see me

cramforce commented on ads/_config.js:
```diff
@@ -34,6 +34,11 @@ export const adPrefetch = {
   revcontent: 'https://labs-cdn.revcontent.com/build/amphtml/revcontent.amp.min.js',
   teads: 'https://cdn.teads.tv/media/format/v3/teads-format.min.js',
+  imobile: [
+    'https://spamp.i-mobile.co.jp/script/amp.js',
```
If you load all of these, why not just load one file?

cramforce commented on ads/_config.js:
```diff
@@ -80,6 +85,10 @@ export const adPreconnect = {
     'https://img.revcontent.com',
   ],
+  imobile: [
+    'https://spad.i-mobile.co.jp',
```
No need to repeat hosts mentioned above.

cramforce commented on ads/imobile.js:
```diff
@@ -0,0 +1,25 @@
+ * @param {!Object} data
+ */
+export function imobile(global, data) {
+  	global.imobileParam = data;
```
nit: indent only 2 spaces

cramforce commented:
Looks good. Some small comments. Please rebase against master and squash commits.

imobile commented:
@cramforce Thank you very much for your quick reply. We have fixed the problems issued by you and squashed the commits.  Looking forward to hearing from your feedback soon, thanks again.

cramforce commented:
Looks great. This is ready to merge, but please rebase against master and repush. There are currently merge conflicts. I believe one of the files you edited was deleted.

cramforce commented:
Please squash commits.

imobile commented:
@cramforce  Sorry we messed something up. We will create a new repo and resend out pull request ASAP. Sorry for the inconvenience.
